health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/component/AcquiredCheck.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.exercisesamplecompose.presentation.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter.Companion.tint
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.wear.compose.material.MaterialTheme
+import com.example.exercisesamplecompose.R
+
+@Composable
+fun AcquiredCheck() {
+    Image(
+        painterResource(id = R.drawable.ic_baseline_check_circle_outline_24),
```

Comment:
You can avoid needing to include drawable resources in code by using the material icons library which then allows you to use an imageVector, and in this case specify Icons.Default.CheckCircleOutline for example.

Reply:
Done